### ISSUE TITLE
support stopping a lxd_instance via terraform

### ISSUE THREAD

**Issue by matifali** [terraform-lxd/terraform-provider-lxd#409]
Support stopping an lxd_instance.
It could be done via a new property.
```hcl
resource "lxd_instance" "demo" {
	...
	state = "running|stopped"
	...
}
```

**Comment by MusicDin** [comment]
Hi,

this is already implemented just not released yet. It will be available in `2.0` like:
```hcl
resource "lxd_instance" "demo" {
	running = "true|false"
	...
}
```

**Comment by matifali** [comment]
@MusicDin Awesome. Thank you :)
I am working on a [Coder](https://github.com/coder/coder) [template](https://registry.coder.com/templates) with an LXD provisioner.
